example/build.gradle

```diff
@@ -30,11 +30,6 @@ android {
     }
 
     buildTypes {
-        debug {
-            minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android.txt')
-        }
```

Comment:
It's mostly a sanity check for myself while working with debug builds, checking that the consumer ProGuard files are doing their job appropriately. In general I like to match ProGuard configurations between debug and release builds anyway, to minimize fun runtime release-only surprises.

BugShaker uses reflection internally to decide whether or not the embedding application depends on Google Play Services. This avoids needing to compile Play Services Maps classes into BugShaker. If ProGuard was not configured properly, the class names used to make this determination would be obscured and the library would fail to (attempt to) capture maps in screenshots. See https://github.com/stkent/bugshaker-android/blob/b93fe8455a41b2f696d48794832754a4d48a6f44/bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java#L281-L284 for the actual check. There may well be a better/more robust way to perform this check, in which case, I'm all ears!